Makefile

```diff
@@ -72,10 +72,27 @@ TAG ?= latest
 CONTROLLER_GEN="${GOBIN}/controller-gen"
 CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
 
+# The CONTAINER_ENGINE variable is used for specifying the container engine. By default 'docker' is used
+# # but this can be overridden when calling make, e.g.
+# # CONTAINER_ENGINE=podman make container-image
+CONTAINER_ENGINE ?= docker
+IMAGE_REGISTRY ?= quay.io/shipwright-io
+SHIPWRIGHT_VERSION = 0.6.0
+IMAGE_VERSION = $(shell scripts/hash-files.sh Dockerfile Makefile | cut -c 1-12)
+CONTAINER_IMAGE = $(IMAGE_REGISTRY)/shipwright-builder-image:v$(SHIPWRIGHT_VERSION)-$(IMAGE_VERSION)
```

Comment:
I think we should only use latest as tag here.

Reply:
Just a side note to raise any amibguity: This one is only local tag on which we append a checksum suffix to detect local file changes and reuse or not the locally built image.
I will set it to latest then and keep the appended suffix. wdyt?